### DISCUSSION TITLE
Is 'vendor' still necessary?

### DISCUSSION THREAD

**Discussion by werbenhu** [mochi-mqtt/server#313] [General]
@mochi-co @dgduncan Since we now require Go version 1.21, do we still need to include the 'vendor' directory in the repository?

**Comment by dgduncan** [comment]
How come you closed this? I actually have never thought about the reason why we vendor.

**Reply by bddckr** [reply]
It seems to be moved to a proper issue in the server repo instead of a discussion here on the org page: https://github.com/mochi-mqtt/server/issues/314
